src/components/SimpleHeader/component.js

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { Text, TouchableOpacity, View } from 'react-native';
+import { styles } from './styles';
+
+const noop = () => {};
+
+export const SimpleHeader = (props) => {
+  const LeftWrapper = props.leftAction ? TouchableOpacity : View;
+  const TitleWrapper = props.titleAction ? TouchableOpacity : View;
+  const RightWrapper = props.rightAction ? TouchableOpacity : View;
+
+  const leftAction = props.leftAction ? () => props.leftAction() : noop;
```

Comment:
I think we can get rid of `noop` and pass null, since `View` doesn't have `onPress` prop.

Created a snack to be sure: https://snack.expo.io/Sy4V5PVc-